libvast/test/parse_data.cpp

```diff
@@ -14,7 +14,7 @@
 #include "vast/concept/parseable/to.hpp"
 #include "vast/concept/parseable/vast/data.hpp"
 
-#define SUITE parseable
+#define SUITE parse_data 
```

Comment:
I would say let's keep this either in suite `parseable` or suite `data`, but I don't think we need to open a separate suite here.

Reply:
Yep, you are right. `parseable` is the better name.